command/catalog_datacenters.go

```diff
@@ -0,0 +1,68 @@
+package command
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/mitchellh/cli"
+)
+
+var _ cli.Command = (*CatalogDatacentersCommand)(nil)
```

Comment:
@kyhavlov I found this on the Internets as a better way to test is something implements an interface. We used to do this in test, but this will fail at build time if the interfaces don't match.